tools/refguide_check.py

```diff
@@ -55,13 +57,23 @@ def get_all_dict(module):
         except ValueError:
             pass
 
+    # FIXME: shouldn't modules be in the refguide, actually? if they're in __all__?
     # somehow some modules survive the first iteration (?)
     for _ in range(2):
         for name in all:
             if inspect.ismodule(getattr(module, name)):
                 all.remove(name)
 
-    return all
+    deprecated = []
+    for name in all:
```

Comment:
Using a built-in as variable name is very ugly (my fault), can you rename it to ``all_dict`` or something like that?